src/lib/file.test.js

```diff
@@ -10,9 +9,7 @@ describe('file.js', () => {
   describe('parsePath()', () => {
     it('should be correct when given `name`', () => {
       const { fullPath, projName, userPath } = parsePath('/Users/Foo/', 'name');
-      const expectedFullPath = isWindows
-        ? '\\Users\\Foo\\name'
-        : '/Users/Foo/name';
+      const expectedFullPath = path.join('/Users', 'Foo', 'name');
```

Comment:
Tested on Windows too? Noticing the included forward slash here and Line 23

Reply:
Yup, this has been tested on Windows. The forward slash is because `path.join` does not include the beginning slash and we do in fact want that.